api/openapi-spec/swagger.json

```diff
@@ -6363,6 +6363,28 @@
      }
     }
    },
+   "v1.AccessCredential": {
+    "description": "AccessCredential represents an public authentication that can be used to authorize remote access to the vm guest",
```

Comment:
an --> a

Reply:
ah, thanks for pointing that out. This actually doesn't read the way I intended. I need to revise this entire sentence.